nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -436,9 +436,6 @@ static void phNciNfc_ResetNtfDelayCb(uint32_t dwTimerId, void *pContext)
     if (NULL != pNciContext)
     {
         (void)phOsalNfc_Timer_Stop(pNciContext->dwNtfTimerId);
-        (void)phOsalNfc_Timer_Delete(pNciContext->dwNtfTimerId);
-
-        pNciContext->dwNtfTimerId = 0;
```

Comment:
dwNtfTimerId was created in phNciNfc_ProcessInitRsp and moved to phNciNfc_DelayForResetNtf to handle CORE_RESET_NTF.
Before change "NCI: Handle NCI_RESET_RSP and NCI_RESET_NTF to manage NCI1.x and NCI2.x" that timer was never
deleted but only stopped when receiving a notification...

The mistake i am fixing here is that once we received a NCI_CORE_RESET_NTF, dwNtfTimerId was deleted and never created again to handle future notification timeout.

This patchrecover the situation where dwNtfTimerId is created as soon as we need it (To handle CORE_RESET_NTF after a hardware reset for instance) and kept valid for future notifications as before.

Do you want me to add this in the commit comment ? If so should i send an updated rc3 pull request ?